packages/web5/src/vc-api.ts

```diff
@@ -1,16 +1,79 @@
-import type { Web5Agent } from '@tbd54566975/web5-agent';
+import { Web5Agent } from '@tbd54566975/web5-agent';
+import type { VerifiableCredential } from '../../credentials/src/types.js';
+
+import { getCurrentXmlSchema112Timestamp } from '../../credentials/src/utils.js';
+import { v4 as uuidv4 } from 'uuid';
+import { dataToBlob } from './utils.js';
+import { Record } from './record.js';
+
+import { DwnInterfaceName, DwnMethodName, RecordsWriteMessage, RecordsWriteOptions } from '@tbd54566975/dwn-sdk-js';
+import { RecordsWriteResponse } from './dwn-api.js';
+
+export type VcCreateResponse = RecordsWriteResponse & {
+  vc: VerifiableCredential;
+};
 
 export class VcApi {
-  #agent: Web5Agent;
+  #web5Agent: Web5Agent;
   #connectedDid: string;
 
   constructor(agent: Web5Agent, connectedDid: string) {
-    this.#agent = agent;
+    this.#web5Agent = agent;
     this.#connectedDid = connectedDid;
   }
 
-  async create() {
-    // TODO: implement
-    throw new Error('Not implemented.');
+  // TODO: Add CreateOptions for more robust VC creation
+  async create(credentialSubject: any): Promise<VcCreateResponse> {
+    if (!credentialSubject || typeof credentialSubject !== 'object') {
+      throw new Error('credentialSubject not valid');
+    }
+
+    const vc: VerifiableCredential = {
+      id                : uuidv4(),
+      '@context'        : ['https://www.w3.org/2018/credentials/v1'],
+      credentialSubject : credentialSubject,
+      type              : ['VerifiableCredential'],
+      issuer            : { id: this.#connectedDid },
+      issuanceDate      : getCurrentXmlSchema112Timestamp(),
+    };
+
+    // TODO: Sign VC
+    // const signedVc = this.#web5Agent.sign(vc);
+
+    const { record, status } = await this._writeRecord(vc);
+    return { record, status, vc };
+  }
+
+  private async _writeRecord(vc: VerifiableCredential) {
```

Comment:
This is just the impl to records.write()
Is this ok for now or should we:

1. do web5.dwn.recrods.write()?
2. make a util function in dwn.record package that they can both share?

Reply:
After thinking about it, I think we should be fine using web5.dwn.records.write.. 
After all this is the web5 api and they should be able to depend on each other, the credentials/ package will be stand along and not have anything like that